scipy/io/matlab/mio.py

```diff
@@ -262,6 +262,17 @@ def savemat(file_name, mdict,
     oned_as : {'row', 'column'}, optional
         If 'column', write 1-D NumPy arrays as column vectors.
         If 'row', write 1-D NumPy arrays as row vectors.
+
+    Examples
+    --------
+    >>> from scipy.io import savemat
+    >>> import numpy as np
+    >>> a = np.arange(20)
+    >>> mdic = {"a": a, "label": "experiment"}
```

Comment:
this is an assignment so returns nothing. You should add `>>> mdic` on the line below, that will give the output that's now present

Reply:
I forgot that, thanks.
I'll fix it.